_dashboards/branding.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Custom branding
+nav_order: 200
+---
+
+# Dashboards custom branding
```

Comment:
NIT: Could it be: `OpenSearch Dashboards custom branding`? Since Dashboards is a component within OpenSearch Dashboards.